pyvisa-py/protocols/usbtmc.py

```diff
@@ -221,10 +233,7 @@ def write(self, data):
         :type data: bytes
         """
 
-        try:
-            return self.usb_send_ep.write(data)
-        except usb.core.USBError as e:
-            raise ValueError(str(e))
+        return self.usb_send_ep.write(data)
```

Comment:
This method should actually never raise an exception so we should handle this in a more elegant manner. But this can be left another pull request.

Reply:
Well, it can of course raise an exception for example if somebody pulls the USB plug.
I changed it because I object to the relabeling as ValueError in principle, and because ValueError is objectively the wrong type of exception.

But I understand my version may not be ideal either. If you dislike this change I can drop it.

Reply:
My point here is that sessions should never leak custom exception but return the proper VISA error code or raise an appropriate Visa exception (the goal is to match the results of using the shared library). But this is not well enforced throughout the code at the moment and fixing it goes beyond the scope of this PR.

Reply:
Clear. Since my change did not really fix it, I put it back the way it was.